src/test/workspaces.test.ts

```diff
@@ -0,0 +1,22 @@
+import * as assert from 'assert';
+import * as vscode from 'vscode';
+import { getDocUri, open } from './helper';
+
+suite('workspace folders', () => {
```

Comment:
Sadly, vscode does not support `workspace.updateWorkspaceFolders` in tests, only caught a glimpse of an error dialog

<img width="474" alt="Screen Shot 2020-10-21 at 3 37 26 PM" src="https://user-images.githubusercontent.com/36832285/96777696-a3537d00-13b8-11eb-9755-930d0304f429.png">

Worse, this test asserts that the result of the call is true (when it wasn't.. sort of) and that there is now 2 workspace folders. Manually testing seemed to reveal that the important event `onDidChangeWorkspaceFolders` doesn't fire? Therefore test passes with `clients.size == 1` but for the wrong reasons, if you wrote a test with 2 independant folders, where we expect `clients.size == 2` I suspect it would fail.

I would leave the testfile in the PR, but comment out the test with a summary, maybe one day we can re-enable it. Sorry for harping on this and then having it be unsupported 😞